server/metrics/statsd.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import (
+	"io"
+	"strings"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+	"github.com/runatlantis/atlantis/server/logging"
+	"github.com/uber-go/tally"
+	tallystatsd "github.com/uber-go/tally/statsd"
+)
+
+func NewLoggingScope(logger logging.SimpleLogging, statsNamespace string) (tally.Scope, io.Closer, error) {
```

Comment:
This seems to only be used by tests? Can you leave a comment about it?

Reply:
It'll be used by the local server setup as well if it's running in debug mode.